actions/install/action.yml

```diff
@@ -0,0 +1,19 @@
+name: setup
+description: 'Installs SLSA verifier and adds it to your PATH'
+branding:
+  icon: 'package'
+  color: 'blue'
+inputs:
+  verifier-release:
+    description: 'slsa-verifier release version to install'
+    required: false
+    default: 'v1.3.0'
```

Comment:
What do you think of making this option `required: true`?
- We don't need to maintain / update the default on every release. 
- Users will declare the version in their workflow, which is easier for readability / troubleshooting for them
- We'd need to be very careful when updating MAJOR semver, because if we forget to update users may be updating the action to a more recent MAJOR version but get a lower version installed. (Although, this is something we could have a pre-release check)

One drawback I see is that users need to update manually. A compromise would be to support versions like `v1` and we can resolve that to the most recent version, but this sort of violate the principle of pinning :/

What does everyone think?

@ianlewis @asraa

Reply:
Maybe there's a solution I overlooked. If we keep `required: false` and build a pre-release step that verifies the new tag is the same as the default version. By default, the action ref and the default version would be in sync. 

If a user overwrites the version, it's clearly defines in their workflow.

The only limitation is that it's not pinned by hash, but I think this is a general limitation of allowing a version option in general.